lib/flores/random.rb

```diff
@@ -20,6 +20,46 @@
 
 # A collection of methods intended for use in randomized testing.
 module Flores::Random
+  DEFAULT_PORT_RANGE = 1024..65535
+  DEFAULT_PORT_CHECK_TIMEOUT = 1
+  DEFAULT_MAXIMUM_PORT_FIND_TRY = 15
+
+
+  # Try to find the first available port from a range.
+  # It will try 15 times and fails, this should give us some room
+  # if something wrong is going on.
+  def self.port(range = DEFAULT_PORT_RANGE)
+    try = 0
+    while try < DEFAULT_MAXIMUM_PORT_FIND_TRY
```

Comment:
I prefer `Fixnum#times` for this:

```ruby
DEFAULT_MAXIMUM_PORT_FIND_TRY.times do 
  candidate = integer(range)
  return candidate if port_available?(candidate)
end

raise ...
```

No refactor is required for this PR, though.